bundles/automation/org.eclipse.smarthome.automation.module.core/src/main/java/org/eclipse/smarthome/automation/module/core/handler/RunRuleActionHandler.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) 1997, 2016 by ProSyst Software GmbH and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.automation.module.core.handler;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.smarthome.automation.Action;
+import org.eclipse.smarthome.automation.RuleRegistry;
+import org.eclipse.smarthome.automation.handler.ActionHandler;
+import org.eclipse.smarthome.automation.handler.BaseModuleHandler;
+import org.eclipse.smarthome.config.core.Configuration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class is a handler for RunRuleAction module type.
+ * It runs the rules which's UIDs are passed by the 'ruleUIDs' property.
+ * !!! If a rule's status is not IDLE that rule can not run !!!
```

Comment:
I don't think we need `!!!` here in front of the sentence and after it.
A single `!` at the end should be enough. Or use `must` / `must not`.